frontend/malayalam-language-explorer/style.css

```
@@ -0,0 +1,544 @@
+  --malayalam-card-hover: rgba(255, 255, 255, 0.98);
+  --malayalam-text: #1a2e26;
+  --malayalam-text-muted: #4e6b5f;
+  --malayalam-accent-glow: rgba(11, 107, 71, 0.15);
+}
+
+.malayalam-main {
+  min-height: 100vh;
+  padding-top: 72px;
+  background-color: var(--malayalam-dark-bg);
+  color: var(--malayalam-text);
+  font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)